vdom/core.py

```diff
@@ -13,6 +13,13 @@
 
 import os
 import io
+try:
+    from html import escape
+except ImportError:
+    # Python 2.x compatibility
+    import cgi
+    from functools import partial
+    escape = partial(cgi.escape, quote=True)
```

Comment:
I really do like this solution better than ours on nbconvert… I may port this over.